src/index.js

```diff
@@ -226,4 +228,34 @@ function applySesTransforms(browserify) {
 
   browserify.transform(removeHtmlComment, { global: true })
   browserify.transform(changeImportString, { global: true })
+}
+
+function validateConfig(configOverride) {
+  if (typeof configOverride !== 'object') {
+    throw new Error("LavaMoat - Expected config override to be an object")
+  }
+  
+  if (!'resources' in Object.keys(configOverride)) {
```

Comment:
`'resources' in configOverride` or `Object.keys(configOverride).includes('resources')`